### PR TITLE
guide: update links

### PR DESCRIPTION
**Pull request by jorgeorpinel**
> Follow-up to https://github.com/iterative/dvc.org/pull/4011#pullrequestreview-1129386637

- [x] Update existing links (based on latest [changes to redirects-list.json](https://github.com/iterative/dvc.org/pull/4011/files#diff-a6828b19e9794c1fbb6799ea95263fb7487b676d054932b5f6e3b315eb12a6a2))

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
This is basically a multi-file find/replace + auto-formatting. I did a visual check of the [many changes](https://github.com/iterative/dvc.org/pull/4017/files) and they seem right. If the link checker passes, this should be mergeable.

File: content/docs/command-reference/version.md [hunk 1]
```diff
@@ -12,17 +12,17 @@ usage: dvc version [-h] [-q | -v]
...
-| `Workspace directory`                       | Filesystem type (e.g. ext4, FAT, etc.) and drive on which the <abbr>workspace</abbr> is mounted                                                                      |
-| `Repo`                                      | Shows whether we are in a DVC repo and/or Git repo                                                                                                                   |
+| Line                                        | Detail                                                                                                                                                                           |
+| ------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [`DVC version`](#components-of-dvc-version) | Version of DVC, and name of the binary or package manager (`pip`, `conda`, etc.) used to install DVC (along with a Git commit hash in case of a development version)             |
```
Comment (shcheklein) [inline content/docs/command-reference/version.md:17]: hmm, why do we have these changes?

**Review by shcheklein — APPROVED** [review]

File: content/docs/command-reference/version.md [hunk 2]
```diff
@@ -12,17 +12,17 @@ usage: dvc version [-h] [-q | -v]
...
-| `Remotes`                                   | Remote [location types](/doc/command-reference/remote/add#supported-storage-types) configured in the repo (e.g. SSH, S3, Google Drive, etc.)                         |
-| `Workspace directory`                       | Filesystem type (e.g. ext4, FAT, etc.) and drive on which the <abbr>workspace</abbr> is mounted                                                                      |
-| `Repo`                                      | Shows whether we are in a DVC repo and/or Git repo                                                                                                                   |
+| Line                                        | Detail                                                                                                                                                                           |
+| ------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```
Comment (jorgeorpinel) [inline content/docs/command-reference/version.md:16]: These crazy long changes are due to linting: MD tables need to accommodate for the length of the updated links in every row...

**Comment by jorgeorpinel** [comment]
> 6/80 links failed.

Fixed links ☝🏼 

Waiting for updated check...

**Comment by jorgeorpinel** [comment]
All links are passing now. Merging!

**Pull request merged**